oms-hypervReplica-solution/README.md

```diff
@@ -0,0 +1,65 @@
+# Hyper-V Replica Solution for OMS
```

Comment:
we have a certain pattern for the folder structure in this repo. Can you consider to rename your folder to: oms-hvr-solution ?